app/views/application-process.html

```diff
@@ -0,0 +1,94 @@
+{% extends "layouts/main.html" %}
+
+{% set title = "How applications work" %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    href: "/application",
+    text: "Back to application"
+  }) }}
```

Comment:
I think we might need to do something cleverer here as this page is also linked to from the signed-out page at the start, plus will be googleable / linkable etc so there might not be anywhere to go back to...  Or we just remove it, but then users have to use the browser back button? 🤔

Reply:
Hmmm, good point. Could it just be 'Back' and then there's some logic to take them back to whatever pervious page they were on?